File: cs.collector/cs.collector/StatsEntry.fs
module StatsEntry
open System

type StatsEntry() =
    member val Name = null with get, set
    member val Unique = null with get,set
    member val TeamKills = 0 with get,set
    member val Damage = 0 with get,set
    member val Deaths = 0 with get,set
    member val Kills = 0 with get,set
    member val Shots = 0  with get,set
    member val Hits = 0 with get,set
    member val HeadShots = 0 with get,set
    member val BombsDefusions = 0 with get,set
    member val BombsDefused = 0 with get,set
    member val BombsPlants = 0 with get,set
    member val BombsExplosions = 0 with get,set
    member val BodyHits = new ResizeArray<int>() with get,set

File: src/parser/StatsEntry.cs
using System;

namespace ConsoleApplication3
{
    public class StatsEntry
    {
        public StatsEntry()
        {
            bodyHits = new uint[9];
        }

        public String name;
        public String unique;
        public uint tks;
        public uint damage;
        public uint deaths;
        public int kills;
        public uint shots;
        public uint hits;
        public uint hs;
        public uint bDefusions;
        public uint bDefused;
        public uint bPlants;
        public uint bExplosions;
        public uint[] bodyHits;

        public override string ToString()
        {
            return name + " " + damage + " " + kills + " " + deaths + " " + shots;
        }
    }
}